### DISCUSSION TITLE
Nemo - tabulator to rename the next file

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#403] [Ideas]
On Windows File Explorer, pressing "Tab" while renaming a file applies the file name to the current file and jumps to the next file in one step. This is very convenient for renaming many files.

Currently, to rename the next file in the list, one needs three keystrokes: enter, arrow down, F2. With tabulator, only one keystroke would be needed instead of three.

Please add this ability to Nemo. Thanks.

**Comment by Jopp-gh** [comment]
Why this hassle if you can select multiple items and rename them in batch? 

You don't have to do renaming manually, check Nemo's shortcuts

**Reply by Florianski** [reply]
the batch renamer is great but it doesnt work in every situation because its not always a simple find&replace operation. Sometimes you really need to manually rename a lot of files and in that case the proposed solution with the tab key would make life a lot easier.

**Comment by 100savage** [comment]
Have you looked at the file renamer app on linux mint? It lets change the names of multiple files at the same time.

**Reply by Florianski** [reply]
the batch renamer is great but it doesnt work in every situation because its not always a simple find&replace operation. Sometimes you really need to manually rename a lot of files and in that case the proposed solution with the tab key would make life a lot easier.
